app/controllers/api/v1/answers_controller.rb

```diff
@@ -2,7 +2,8 @@ module Api
     module V1
         class AnswersController < ApplicationController
             def index
-                answers = Answer.all
+                # answers = Answer.all
+                answers = Answer.where(query_params)
```

Comment:
I'm only able to search by question_id and user_id, what if I want to filter by user name, email or by question label?

Reply:
> I'm only able to search by question_id and user_id, what if I want to filter by user name, email or by question label?

I the last commit. I implement search by foreign fields, just fallow this format 

"users.name": "Yoher Obando Duarte"
"users.email": "yoher.od96@hotmail.com"

"questions.label": "What is the capital of France?"
"questions.active": true or false